toolkit/tests/pastaporto/test_tokens.py

```diff
@@ -72,3 +75,19 @@ async def _():
         decode_service_to_service_token(
             test_token, "secret", issuer="test", audience="association-backend"
         )
+
+
+@test("generate a token")
+async def _():
+    token = generate_service_to_service_token("secret", issuer="me", audience="you")
+
+    assert token
+
```

Comment:
This test doesn't seem very useful, should we test it against a known token? Maybe freeze the time with time_machine